frontend/src/services/matchingService.js

```diff
@@ -0,0 +1,24 @@
+import axios from "axios"
+const baseUrl = 'http://localhost:8001'
+
+const findMatch = async (matchObject) => {
+    const res = await axios.post(`${baseUrl}/find-match`, matchObject)
+    return res
+}
+
+const cancelFindMatch = async (matchObject) => {
+    const res = await axios.delete(`${baseUrl}/cancel-find-match`, { data: matchObject })
+    return res
+}
+
+const getInterview = async (interviewId) => {
+    const res = await axios.get(`${baseUrl}/interview/${interviewId}`)
+    return res
+}
+
+const deleteInterview = async (interviewId) => {
+    const res = await axios.delete(`${baseUrl}/end-interview/${interviewId}`)
+    return res
```

Comment:
Can consider 1 line return statement instead? 

Not sure if we need try-catch block https://stackoverflow.com/questions/7148019/when-should-you-use-try-catch-in-javascript

but maybe can add if statement checks instead just incase of a null response in that case? (means no more 1 line return statement)

Reply:
Should backend do the null checking or frontend do the null checking?

Reply:
Both would be good.